ChildProject/annotations.py

```diff
@@ -0,0 +1,306 @@
+        IndexColumn(name = 'imported_at', description = 'importationd date', datetime = "%Y-%m-%d %H:%M:%S", required = False),
+        IndexColumn(name = 'format', description = 'input annotation format', regex = r"(TextGrid|eaf|rttm)", required = True)
+    ]
+
+    SEGMENTS_COLUMNS = [
+        IndexColumn(name = 'annotation_file', description = 'raw annotation path relative to /raw_annotations/', required = True),
+        IndexColumn(name = 'segment_onset', description = 'segment start time in seconds', regex = r"[0-9]{1,}(\.[0-9]{3})?", required = True),
+        IndexColumn(name = 'segment_offset', description = 'segment end time in seconds', regex = r"[0-9]{1,}(\.[0-9]{3})?", required = True),
+        IndexColumn(name = 'speaker_id', description = 'identity of speaker in the annotation', required = True),
+        IndexColumn(name = 'speaker_type', description = 'class of speaker (FEM, MAL, CHI, OCH)', required = True),
+        IndexColumn(name = 'ling_type', description = '1 if the vocalization contains at least a vowel (ie canonical or non-canonical), 0 if crying or laughing', required = True),
+        IndexColumn(name = 'vcm_type', description = 'vocal maturity defined as: C (canonical), N (non-canonical), Y (crying) L (laughing), J (junk)', required = True),
+        IndexColumn(name = 'lex_type', description = 'W if meaningful, 0 otherwise', required = True),
+        IndexColumn(name = 'mwu_type', description = 'M if multiword, 1 if single word -- only filled if lex_type==W', required = True),
+        IndexColumn(name = 'addresseee', description = 'T if target-child-directed, C if other-child-directed, A if adult-directed, U if uncertain or other', required = True),
+        IndexColumn(name = 'transcription', description = 'orthographic transcription of the speach', required = True)
+    ]
+
+    SPEAKER_ID_TO_TYPE = {
+        'C1': 'OCH',
+        'C2': 'OCH',
+        'CHI': 'CHI',
+        'CHI*': 'CHI',
+        'EE1': 'ELE',
+        'FA0': 'FEM',
+        'FA1': 'FEM',
+        'FA2': 'FEM',
+        'FA3': 'FEM',
+        'FA4': 'FEM',
+        'FA5': 'FEM',
+        'FA6': 'FEM',
+        'FA7': 'FEM',
+        'FA8': 'FEM',
+        'FAE': 'ELE',
+        'FC1': 'OCH',
+        'FC2': 'OCH',
+        'FC3': 'OCH',
+        'MA0': 'MAL',
+        'MA1': 'MAL',
+        'MA2': 'MAL',
+        'MA3': 'MAL',
+        'MA4': 'MAL',
+        'MA5': 'MAL',
+        'MAE': 'ELE',
+        'MC1': 'OCH',
+        'MC2': 'OCH',
+        'MC3': 'OCH',
+        'MC4': 'OCH',
+        'MC5': 'OCH',
+        'MI1': 'OCH',
+        'MOT*': 'FEM',
+        'OC0': 'OCH',
+        'UC1': 'OCH',
+        'UC2': 'OCH',
+        'UC3': 'OCH',
+        'UC4': 'OCH',
+        'UC5': 'OCH',
+        'UC6': 'OCH'
+    }
+
+    VTC_SPEAKER_TYPE_TRANSLATION = defaultdict(lambda: 'NA', {
+        'CHI': 'OCHI',
```

Comment:
line 82 should map CHI to OCH (not OCHI)

Reply:
fixed in 0d18ce91d916924188d0b1cc2b4ea048bbc893e4